lib/exchange_rate.py

```diff
@@ -328,6 +327,11 @@ def get_rates(self,ccy):
     def history_ccys(self):
         return ['BRL']
```

Comment:
While at it, could you also remove this `history_ccys` method for `NegocieCoins` too? :)
It is not useful without a `request_history` method.